### ISSUE TITLE
[MRESOLVER-34] Update dependences and plugins to latest versions

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-resolver#811]
**[Sylwester Lachiewicz](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=slachiewicz)** opened **[MRESOLVER-34](https://issues.apache.org/jira/browse/MRESOLVER-34?redirect=false)** and commented

Update to latest versions based on versions:display-dependency-updates report

```
org.apache.httpcomponents:httpcore .................... 4.4.6 -> 4.4.8
org.eclipse.jetty:jetty-server .... 9.2.9.v20150224 -> 9.4.8.v20171121
org.eclipse.jetty:jetty-servlet ... 9.2.9.v20150224 -> 9.4.8.v20171121
org.eclipse.jetty:jetty-util ...... 9.2.9.v20150224 -> 9.4.8.v20171121
org.codehaus.plexus:plexus-utils ..................... 3.0.24 -> 3.1.0
org.eclipse.sisu:org.eclipse.sisu.plexus .............. 0.1.1 -> 0.3.3
org.sonatype.sisu:sisu-guice .......................... 3.1.6 -> 3.2.6
```

And plugins in demos:
maven-invoker-plugin:3.0.1
junit:junit:4.12
org.eclipse.sisu.plexus:0.3.3

---
No further details from [MRESOLVER-34](https://issues.apache.org/jira/browse/MRESOLVER-34?redirect=false)

**Comment by jira-importer** [comment]
**[Sylwester Lachiewicz](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=slachiewicz)** commented

enforcer java 1.7 rule - latest java 7 jetty - 9.2.22.v20170606

**Comment by jira-importer** [comment]
**[Sylwester Lachiewicz](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=slachiewicz)** commented

https://github.com/apache/maven-resolver/pull/11

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with [6f3158e6f62eb75ea9f2de857ba9babc0c174157](https://git-wip-us.apache.org/repos/asf?p=maven-resolver.git;a=commit;h=6f3158e6f62eb75ea9f2de857ba9babc0c174157).
